### ISSUE TITLE
Can't start the standalone client

### ISSUE THREAD

**Issue by u9g** [extremeheat/prismarine-inspector#2]
I ran `npm install -g prismarine-inspector` and it installed fine, but then running `npx prismarine-inspector` just hangs the terminal, any ideas?

**Comment by u9g** [comment]
Running manually from node_modules, it throws with 
```
Error: Cannot find module './back/app'
Require stack:
- C:\Users\Jason\AppData\Roaming\npm\node_modules\prismarine-inspector\src\main.js
- C:\Users\Jason\AppData\Roaming\npm\node_modules\prismarine-inspector\node_modules\electron\dist\resources\default_app.asar\main.js
-
    at Module._resolveFilename (internal/modules/cjs/loader.js:717:15)
    at Function../lib/common/reset-search-paths.ts.Module._resolveFilename (electron/js2c/browser_init.js:7729:16)
    at Module._load (internal/modules/cjs/loader.js:622:27)
    at Module._load (electron/js2c/asar.js:717:26)
    at Function.Module._load (electron/js2c/asar.js:717:26)
    at Module.require (internal/modules/cjs/loader.js:775:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (C:\Users\Jason\AppData\Roaming\npm\node_modules\prismarine-inspector\src\main.js:3:13)
    at Module._compile (internal/modules/cjs/loader.js:880:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:892:10)
```

**Comment by extremeheat** [comment]
Released 0.1.5 which should fix the standalone client
